### ISSUE TITLE
Access RTDE variables 

### ISSUE THREAD

**Issue by InesRis** [UniversalRobots/Universal_Robots_ROS_Driver#315]
I am new in ROS and currently involved in a project with an UR3e.
I need to access to the RTDE data.  I have couple of questions I would be super grateful if you´d help me clarify:

It seems that the variables that I need to access is stated in the following txt:

/ur_robot_driver/resources/rtde_output_recipe.txt

However, I am not sure how would it be possible for me to access to them from the terminal or from a script. 

This topic is also approached in the issue #35. It seems that what they tried to do here is to publish this data into different topics. Therefore, I would like to know, if the only way of accessing to these variables is through the topics, meaning that I need to install this branch of the driver? If so, after installing this branch of the package, How do the RTDE variables can be seen/printed and what are the specific names of the topics to which these variables are being saved to?

Thanks so much in advance for your time and help!

**Comment by fmauch** [comment]
Yes, #35 would be the way to access arbitrary RTDE data. I'm sorry, this isn't high enough in our priority list currently.

Did [this comment](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/35#issuecomment-768191372) not answer your second question, though?

**Comment by urfeex** [comment]
#35 will not be merged for ROS 1 anymore. ROS 2 will hopefully receive a similar feature in the future (see https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/issues/1648).
